easybuild/easyconfigs/p/python-magic/python-magic-0.4.27-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'python-magic'
+version = '0.4.27'
+
+homepage = 'http://github.com/ahupp/python-magic'
+description = """
+python-magic is a Python interface to the libmagic file type identification library. libmagic
+identifies file types by checking their headers according to a predefined list of file types. 
```

Comment:
this strikes me as a very minor package, perhaps best left to be part of a bundle for whichever software needs it instead?